Bottom: 983a145a01c949fba7379a35e1841c79f3703fd9
Top:    b18b35a8cf3df6b6dd478d7fa28f660b0d817fe7
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-17 22:04:47 +0800

Refresh of virtio-net-optimize-small

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 42cbf9f..6b8d8af 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -448,7 +448,7 @@ static struct sk_buff *receive_small(struct net_device *dev,
 	if (!delta) {
 		buf += VIRTNET_RX_PAD + virtnet_get_headroom(vi);
 		memcpy(skb_vnet_hdr(skb), buf, vi->hdr_len);
-	} /* keep zeored vnet hdr since packet was changed by bpf */
+	} /* keep zeroed vnet hdr since packet was changed by bpf */
 
 err:
 	return skb;
